tests/src/test/scala/whisk/core/containerpool/test/ContainerProxyTests.scala

```diff
@@ -761,7 +761,7 @@ class ContainerProxyTests
     override def run(parameters: JsObject, environment: JsObject, timeout: FiniteDuration)(
       implicit transid: TransactionId): Future[(Interval, ActivationResponse)] = {
       runCount += 1
-      environment.fields("api_key") shouldBe message.user.authkey.toJson
+      environment.fields("api_key") shouldBe message.user.authkey.toEnvironment.fields("api_key")
```

Comment:
Should this check if `environment` contains all keys + values from `message.user.authkey.toEnvironment` for completeness?

Reply:
I have added a test for that